src/Containers/Home/Home.js

```diff
@@ -25,7 +26,7 @@ class Home extends React.Component {
   render () {
     return (
       <div className='grid contentbox'>
-        <div className='divrecentbox pad15'>
```

Comment:
We should do background images from the SCSS as done before rather than adding two types of style via inline and SCSS classes